src/Microsoft.OData.Edm/Validation/ValidationRuleSet.cs

```diff
@@ -102,6 +102,7 @@ public sealed class ValidationRuleSet : IEnumerable<ValidationRule>
                 ValidationRules.PropertyValueBindingValueIsCorrectType,
                 ValidationRules.EnumMustHaveIntegerUnderlyingType,
                 ValidationRules.AnnotationInaccessibleTerm,
+                ValidationRules.VocabularyAnnotationTargetAllowedApplyToElement,
                 ValidationRules.ElementDirectValueAnnotationFullNameMustBeUnique,

```

Comment:
I thought we were not going to add this rule to the default ruleset, since appliesto is only a hint and clients need to be prepared to handle a term being applied to other elements?

Reply:
#Resolved.